packages/requisitions/src/ResponseRequisition/DetailView/AppBarButtons/CreateShipmentButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  ButtonWithIcon,
+  PlusCircleIcon,
+  useTranslation,
+  useConfirmationModal,
+  useAlertModal,
+} from '@openmsupply-client/common';
+import { useCreateOutboundFromResponse, useResponseFields } from '../../api';
+
+export const CreateShipmentButtonComponent = () => {
+  const { linesRemainingToSupply } = useResponseFields(
+    'linesRemainingToSupply'
+  );
+  const t = useTranslation('distribution');
+  const { mutate: createOutbound } = useCreateOutboundFromResponse();
+  const getConfirmation = useConfirmationModal({
+    onConfirm: createOutbound,
+    message: t('messages.create-outbound-from-requisition'),
+    title: t('heading.create-outbound-shipment'),
+  });
+  const alert = useAlertModal({
+    title: t('heading.cannot-do-that'),
+    message: t('message.all-lines-have-been-fulfilled'),
+  });
+
+  const onCreateShipment = () => {
+    if (linesRemainingToSupply.totalCount > 0) {
+      alert();
+    } else {
+      getConfirmation();
+    }
```

Comment:
It did make testing easier! should probably reverse the logic now though 😄 
```suggestion
    if (linesRemainingToSupply.totalCount > 0) {
      getConfirmation();
    } else {
      alert();
    }
```

Reply:
haha whoops!

I just tested and I am getting the same error on the demo server. However, not on my local server 🤔